### PR TITLE
Mma7361

### PR DESCRIPTION
**Pull request by BrianGenisio**
This PR adds [support for the MMA-7361](https://github.com/rwaldron/johnny-five/issues/564).  This device is different from others in that the three axis have different center values which can vary from device, and it has a "sleep" pin that when set to low, puts the device in sleep mode.

- Support the MMA-7361 with optional `sleepPin`
- Allow `zeroV` to be either an `Array` or `Number`. Available to any device type.
- Support an `autoCalibrate` boolean which will use the first several reads to find the center values.  Available to any device type.
- Export the `zeroV` property on the device so that it can be read and remembered for later (used in init)
- Add an `enable()` and `disable()` function that will turn the device on or off (stop responding to data events).  If the MMA-7361 provides a `sleepPin`, set the pin low to put the device to sleep.

**TODO**: ~~Fritzing diagram~~, ~~unit tests~~, and Wiki documentation updates (after it lands)

### CODE REVIEW

File: lib/accelerometer.js [hunk 1]
```diff
@@ -242,16 +305,38 @@ function Accelerometer(opts) {
     this.toGravity = opts.toGravity || function(raw) { return raw; };
   }
 
+  if (!this.onEnable) {
+    this.onEnable = function(on) {};
```
Comment (rwaldron) [inline lib/accelerometer.js]: Make this `enable` with a `disable` counterpart.

Comment (rwaldron) [inline lib/accelerometer.js]: Also, you can fake "enable"/"disable" for those that don't have hardware support by just returning from the dataHandler if state.enabled === false

Comment (BrianGenisio) [inline lib/accelerometer.js]: One step ahead of you on that one :)

**Comment by BrianGenisio** [comment]
I can't find a Fritzing component for this.  Is there a suggested approach to create a fritzing diagram for something like this that doesn't have a component?

**Comment by rwaldron** [comment]
I make the art (svg) in iDraw and then use an existing component (which is dumb, but that's how Fritzing works) to create a new component

**Comment by BrianGenisio** [comment]
Awesome.  I'm done with this PR, pending review and secondary validation.

**Comment by rwaldron** [comment]
There's a merge conflict in programs.json

File: eg/accelerometer-mma7361.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  // This device also benefits from a calibration step.  You can autoCalibrate
+  // by placing the device level on startup.  You can also read the calibrated
+  // centers by reading the accelerometer.zeroV array after calibration.  Subsequent
+  // initializations, you can omit the autoCalibrate and set the zeroV array
+  // in the options instead
+
+  var accelerometer = new five.Accelerometer({
+    controller: "MMA7361",
+    pins: ["A0", "A1", "A2"],
+    sleepPin: 13,
```
Comment (rwaldron) [inline eg/accelerometer-mma7361.js]: I just realized this will be missed by the `Board.Component` checks for "currently occupied pins" :\

**Comment by rwaldron** [comment]
Landed https://github.com/rwaldron/johnny-five/commit/fb02c8953a04fedb0712c3a839d7b366fa901dd5

**Pull request closed without merging**